src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -1,267 +1,17 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System;
-using System.Linq;
-using System.Security;
-using System.Threading;
 using System.Threading.Tasks;
 using System.Xml.Linq;
 using Microsoft.CodeAnalysis.CSharp;
-using Microsoft.CodeAnalysis.Editor.UnitTests.QuickInfo;
 using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
 using Microsoft.CodeAnalysis.Test.Utilities;
-using Microsoft.CodeAnalysis.QuickInfo;
 using Roslyn.Test.Utilities;
-using Roslyn.Utilities;
 using Xunit;
-using static Microsoft.CodeAnalysis.Editor.UnitTests.Classification.FormattedClassifications;
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.QuickInfo
 {
-    public partial class SemanticQuickInfoSourceTests : AbstractSemanticQuickInfoSourceTests
+    public sealed class SemanticQuickInfoSourceTests : SemanticQuickInfoSourceTestsBase
```

Comment:
The change in this commit (9d954cb697a34d565480bebb81f973035c287cf2) - separating parts of a test class into separate classes while extracting the common helpers into a base is a really good use case for #31837. I got a bunch of accessibility errors because I needed to change each of those helpers from private to protected and had to go through them one by one by hitting F12 and changing the modifier.

Reply:
This might even be a reason to have a fix all for that.